httplib2_transport/tox.ini

```diff
@@ -0,0 +1,38 @@
+[tox]
+envlist = lint,py27,py34,py35,pypy,cover
+
+[testenv]
+deps =
+  flask
+  mock
+  pytest
+  pytest-cov
+  pytest-localserver
+  httplib2
+  {toxinidir}/..
```

Comment:
This masks the fact that the `httplib2` package doesn't depend on the main one in `setup.py` (but you still want this in `deps`)